org.jaspstats.JASP.json

```diff
@@ -0,0 +1,4046 @@
+{
+	"app-id": "org.jaspstats.JASP",
+	"runtime": "org.kde.Sdk",
+	"runtime-version": "5.12",
+	"sdk": "org.kde.Sdk",
+	"command": "org.jaspstats.JASP",
+	"inherit-extensions" : [ "org.freedesktop.Platform.GL" ],
+	"finish-args":[
+		"--socket=x11",
+		"--socket=wayland",
+		"--share=ipc",
+		"--share=network",
+		"--filesystem=home",
+		"--env=PATH=/usr/bin:/app/bin",
```

Comment:
Why you would need this?

Reply:
which part exactly? 
The `PATH`? We start a sub-executable which needs to be in the PATH.
ipc was necessary for X11 as far as I understood from the docs, network is necessary because we support reading and writing to [OSF](https://osf.io/).
Home is needed to read and write stuff there, no?

Reply:
I meant the PATH. The sub-executable is shipped with the flatpak?

Reply:
Yeah, it is next to the main executable (main: `jasp` and sub: `JASPEngine`)

Reply:
It's quite some time ago I put that in and if it isn't necessary or wrong I can of course change it.

Reply:
yeah,  it's weird to mess with the PATH. Better try without first